lexers/p/python_test.go

```diff
@@ -0,0 +1,20 @@
+package p_test
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/alecthomas/assert"
+	"github.com/alecthomas/chroma"
+	"github.com/alecthomas/chroma/lexers/p"
+)
+
+func TestPython3_AnalyseText(t *testing.T) {
```

Comment:
As lexer is named `Python`, let's have the test name aligned here. Pip finally ending support for Python2. Python3 is the new standard eventually. This took some years :joy: 
```suggestion
func TestPython_AnalyseText(t *testing.T) {
```

Reply:
I think that's a decision to make!